core/wiki/macros/timeline.tid

```diff
@@ -3,22 +3,19 @@ modified: 20141212110330815
 tags: $:/tags/Macro
 title: $:/core/macros/timeline
 
-\define timeline-title()
-\whitespace trim
-<!-- Override this macro with a global macro 
-     of the same name if you need to change 
-     how titles are displayed on the timeline 
-     -->
-<$view field="title"/>
-\end
+<!-- Override one or both of the following two macros with a global or local macro of the same name 
```

Comment:
I don't think one can override a macro definition from the same tiddler with a global macro definition. Does it definitely work?

Reply:
Did just test this. It definitely works with the changes here, since user macros are initialized after core macros. I'll need to test the other PR